test

```diff
@@ -87,8 +87,12 @@ docker kill jupyter_test_r && docker rm jupyter_test_r
 docker run --rm --name=papermill_test_r --read-only --net=none \
     "$IMAGE_TAG" python -c 'import sys;import papermill as pm; print(pm.__version__)'
 
+
+# TF_FORCE_GPU_ALLOW_GROWTH is to prevent tensorflow from allocating the totality of a GPU memory.
```

Comment:
👍  We do the same for our Python image: https://github.com/Kaggle/docker-python/blob/ce2a5245a1e2481e7384caec03feafe38bac24da/test#L105

Reply:
Oh nice, I wasn't aware of that!